packages/core/js-client/src/jsPeer/FluencePeer.ts

```diff
@@ -560,17 +559,16 @@ export abstract class FluencePeer {
         this._timeouts.forEach((timeout) => {
             clearTimeout(timeout);
         });
-        this._particleQueues.clear();
     }
 }
 
-function filterExpiredParticles(onParticleExpiration: (item: ParticleQueueItem) => void) {
+function filterExpiredParticles<T extends ParticleQueueItem = ParticleQueueItem>(onParticleExpiration: (item: T) => void) {
```

Comment:
Is this default really needed? I almost never used it, I think it's need when e.g. some argument is optional, but we always have item in the code so seems like we can remove this default like this:
```suggestion
function filterExpiredParticles<T extends ParticleQueueItem>(onParticleExpiration: (item: T) => void) {
```